internal/clients/http/ec2/ec2_client.go

```diff
@@ -442,6 +485,52 @@ func getRoleName(arn string) (string, error) {
 	return roleName[1], nil
 }
 
+func getStatementFromAWSDocument(document string) ([]Statement, error) {
+	unescapedDoc, err := url.QueryUnescape(document)
+	if err != nil {
+		return nil, fmt.Errorf("Could not parse url from the Document: %w", err)
```

Comment:
All errors must start with lowercase (our linter should enforce it not sure why it passes). On the other hand, logs start with capital letter. If I could change this when I started with capitals I would do this. The latter is not enforced tho.